pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -176,7 +176,7 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
         if ftype == 'pinhole' or ftype == 'dark' or ftype == 'bias':
             # Don't type pinhole, dark, or bias frames
             return np.zeros(len(fitstbl), dtype=bool)
-        if ftype == 'arc':
+        if ftype == 'arc' or ftype == 'tilt':
```

Comment:
FWIW, I prefer:

```
if ftype in ['arc', 'tilt']
```

makes it easy to keep adding.

Reply:
Agreed.  Changed.  Although I (irrationally?) dislike, e.g., `if ftype in ['arc']:`